<?xml version="1.0" encoding="utf-8"?>
<launch>
  <rosparam file="$(find dynamixel_control)/config/config.yaml" />
  <node name="joint_control" pkg="dynamixel_control" type="joint_control" output="screen"/>
</launch>
